js/reference.js

```diff
@@ -473,6 +494,43 @@ var ERMrest = (function(module) {
                         this._referenceColumns.push(compositeFKs[i]);
                     }
                 }
+
+                // if edit context remove filename, bytecount, md5, and sha256 from visible columns
+                if (module._isEntryContext(this._context) && assetColumns.length !== 0) {
+                    // given a colName will remove it from visible columns
+                    var removeCol = function(col) {
+                        // property does not exist
+                        if (col === null) {
+                            return;
```

Comment:
This seems like a case where the API is being used incorrectly and should actually throw an error. Maybe I missed something though.